ORK1Kit/ORK1Kit/Common/ORK1StepHeaderView.m

```diff
@@ -132,6 +132,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
 #endif
         [self setUpConstraints];
         [self setNeedsUpdateConstraints];
+        self.layoutMargins = UIEdgeInsetsMake(8, 15, 8, 15);
```

Comment:
ORK1StepHeaderView contains the step title and detail text. Previously this was the default 8px, here we make it 15 so the text lines up with the buttons and text choices.